package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -141,7 +140,8 @@ class TRZReader(base.ReaderBase):
        Extra data (Temperature, Energies, Pressures, etc) now read
        into ts.data dictionary.
        Now passes a weakref of self to ts (ts._reader).
-
+    .. versionchanged:: 2.0.0
```

Comment:
I'm pretty sure this won't work, you usually need 2 lines between the text and the versionchanged

Reply:
It's right after a (quite long) `.. versionchanged`:) The doc lgtm.

Reply:
Sorry my bad, I somehow managed to miss the above versionchanged.